cumulus/parachains/common/src/rococo.rs

```diff
@@ -117,3 +117,16 @@ pub mod consensus {
 	/// Relay chain slot duration, in milliseconds.
 	pub const RELAY_CHAIN_SLOT_DURATION_MILLIS: u32 = 6000;
 }
+
+pub mod snowbridge {
+	use frame_support::parameter_types;
+	use xcm::opaque::lts::NetworkId;
+
+	/// The pallet index of the Ethereum inbound queue pallet in the bridge hub runtime.
+	pub const INBOUND_QUEUE_PALLET_INDEX: u8 = 80;
+
+	parameter_types! {
+		/// Network and location for the Ethereum chain.
```

Comment:
@claravanstaden 
nit: could you please add also here some doc/ref_link where we could check `11155111` what it really is?

Reply:
https://github.com/paritytech/polkadot-sdk/pull/2894/commits/a6fdfda2fe03be92a0a6db624e2597ffa977d5e6 :)